coremltools/converters/mil/mil/ops/defs/reduction.py

```diff
@@ -126,6 +156,36 @@ def get_operator(self):
 
 @register_op(doc_str="TODO")
 class reduce_argmin(reduce_arg):
+    """
+    Computes the indices of the minimum value across dimensions of a tensor.
+    In case of ties, the identity of the return value is not guaranteed.
+    
+    Parameters
+    ----------
+    x: <*,T> (Required)
+        * Must be 1-dimensional or higher.
+    
+    axes: const<i32> (Optional)
+        * The dimensions to reduce.
+    
+    keep_dims: const<bool> (Optional, default=False)
+        * If ``False``, the rank is reduced for each entry in ``axes``,
```

Comment:
---> If ``False``, the rank is reduced by 1, by removing the dimensions specified via ``axis``.